snippets/numpy/core/multiarray.py

```diff
@@ -0,0 +1,21 @@
+from ulab import numpy as np
+
+def asarray(a, dtype=None):
```

Comment:
What is the difference between `asarray`, and `array`? Only that `asarray` returns the array, if an `ndarray` is passed, while `array` returns a new copy? We could definitely add this function in C, if you think it is useful.

Reply:
My rough implementation of https://numpy.org/doc/stable/reference/generated/numpy.asarray.html

Changing this to C would be much prefered.

Reply:
https://github.com/v923z/micropython-ulab/issues/480

Reply:
OK, we could add that.